controllers/kyma_controller/manifest_test.go

```diff
@@ -316,7 +316,7 @@ func validateManifestSpecInstallSourceRepo(manifestImageSpec *v1beta2.ImageSpec,
 //nolint:goerr113
 func validateManifestSpecInstallSourceType(manifestImageSpec *v1beta2.ImageSpec) error {
 	actualSourceType := string(manifestImageSpec.Type)
-	expectedSourceType := "oci-ref" //no corresponding value in the ModuleTemplate?
+	expectedSourceType := "oci-ref" // no corresponding value in the ModuleTemplate?
```

Comment:
```suggestion
	expectedSourceType := "string(v1beta2.OciRefType)" // no corresponding value in the ModuleTemplate?
```

Reply:
you should not quote it 😄

Reply:
and actually it's your code, self-review is good practice. 😄